pyproject.toml

```diff
@@ -0,0 +1,115 @@
+[project]
+name = "azure-functions-worker"
+version = "4.29.0"
```

Comment:
Can we reference the version.py file in pyproject.toml? If not, lets remove version.py